mbuild/compound.py

```diff
@@ -2476,8 +2476,8 @@ def to_parmed(self, box=None, title='', residues=None, show_ports=False,
                 if not val:
                     box_vec_max[dim] += 0.25
                     box_vec_min[dim] -= 0.25
-            box.mins = np.asarray(box_vec_min)
             box.maxs = np.asarray(box_vec_max)
```

Comment:
LGTM, can you include either a comment here, or one in the commit message describing this? If we look at this a few months from now I feel like we might forget that the order matters.

Reply:
Added